dm/dm-webui-guide.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Use WebUI to manage DM migration tasks.
+summary: Learn how to use WebUI to manage DM migration tasks.
+---
+
+# Use WebUI to manage DM migration tasks
+
+DM WebUI is web-based GUI platform for managing TiDB Data Migration (DM) tasks. It provides a simple and intuitive way to manage a large number of migration tasks, which frees you from using the dmctl command-line tool.
+
+This document introduces how to access DM WebUI, the prerequisites, the use cases of each interface, and the attention points.
+
+> **Warning:**
+>
+> - DM WebUI is currently an experimental feature. It is not recommended to use it in the production environment.
+> - The lifecycle of `task` in DM WebUI has changed, and it is not recommended to use DM WebUI and dmctl at the same time.
+
+DM WebUI has the following pages:
+
+- **Dashboard**: Displays the main monitoring information and status information of migration tasks in DM to help you quickly learn the overall running status of migration tasks and the key metrics on latency and performance.
+- **Migration**
+    - **Task**: Provides an entry to task creation, and displays the detailed information of each migration task. This page helps you monitor, create, delete, and configure migration tasks.
+    - **Source**: Configures the information of upstream data source for a migration task. On this page, you can manage the upstream configuration in a data migration environment, including creating and deleting upstream configuration, monitoring the task status corresponding to the upstream configuration, and modifying upstream configuration.
+    - **Replication Detail**: Displays the detailed status information of migration tasks. On this page,  you can view the detailed configuration and status information based on a specified filter, including the configuration information and database names of the upstream and downstream.
```

Comment:
`including the configuration information and database names of the upstream and downstream.`

the users can view the actually replication details here. like the relation of source table and target table.

Reply:
This is also a literal translation of the Chinese. Based on your comment, I've modified the Chinese: https://github.com/pingcap/docs-cn/pull/8599/files#r831900928. PTAL. Thanks~
```suggestion
    - **Replication Detail**: Displays the detailed status information of migration tasks. On this page,  you can view the detailed configuration and status information based on a specified filter, including the configuration information and database names of the upstream and downstream, the relation of source tables and target tables.
```